README.md

```diff
@@ -763,6 +765,59 @@ We have a small collection of algorithms, one for every day of the month. Scroll
     - [Introduction to matrices](https://en.wikipedia.org/wiki/Matrix_(mathematics))
 
 
+## **December 29 - Cup and String Phone**
+  - **Problem :**
+    - Tharun is playing with his friends using a string phone. To make this interesting he asked his friends to join him. He asked them to bring cup of their own and he will bring the strings for all of them so that all can communicate with each other. Now he wants to find the number of strings required to communicate with his friends. Help him with a program that takes a number of friends as input and give the number of strings as output.
+
+    For example Tharun plays with 2 of his friends. Then he will bring 3 Strings.This is because 1 string is enough to connect 2 friends.
+
+    <img src="/src/docs/dec29-string-phone.jpg" width="450" height="300">
+
+  - **Examples :**
+  
+    1) Input : 7\
+       Output : 28\
+       Explanation : \
+            Tharun & his 6 friends come together to play this game. So to connect all of 7 cups, we need 28 strings.\
```

Comment:
This isn't really an explanation. This is just stating the input and output in word form.